internal/cmd/launcher_test.go

```diff
@@ -348,3 +350,60 @@ func TestIsAnalysisRequired(t *testing.T) {
 		})
 	}
 }
+
+func TestGetBuildServiceURL(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name                      string
+		buildSrvURL               string
+		enableCommunityExtensions bool
+		expectErr                 bool
```

Comment:
```suggestion
		expectErr                 error
```
Instead of using a boolean here, you can use directly an error. In this way you can remove the condition and directly assert via `require.Equal(t, tc.expectErr, err)`. When the error isn't expcted then it is just set to `nil` like `expectErr: nil`.